modules/test/playwright/tests/commerce/commerce-order-content-web/commerceLayouts.spec.ts

```diff
@@ -2691,3 +2691,31 @@ test('LPD-43496 Account selector redirects to order details DPT if open order co
 
 	await expect(page.getByText('Heading Example')).toBeVisible();
 });
+
+test(
+	'When there is no site associated with the channel, inform the user',
+	{tag: '@LPD-51595'},
+	async ({apiHelpers, commerceLayoutsPage, page}) => {
+		const {channel, site} = await miniumSetUp(apiHelpers);
```

Comment:
This is temporarily ok, but let's start basing our tests on Classic, it should also be ever so slightly faster